clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java

```diff
@@ -208,6 +208,79 @@ public void createServerGroupOperationTest() throws IOException { // no response
     assertNotNull(response); // TODO inspect response contents
   }
 
+  @Test
+  public void getServiceDiscoveryRegistriesTest() {
+    // given
+    ProviderCache ecsCache = providerRegistry.getProviderCache(EcsProvider.NAME);
+    String testClusterName = "spinnaker-deployment-cluster";
+    String testNamespace = Keys.Namespace.SERVICE_DISCOVERY_REGISTRIES.ns;
+    String clusterKey = Keys.getClusterKey(ECS_ACCOUNT_NAME, TEST_REGION, testClusterName);
+    String url = getTestUrl("/ecs/serviceDiscoveryRegistries");
+    Map<String, Object> attributes = new HashMap<>();
+    attributes.put("account", ECS_ACCOUNT_NAME);
+    attributes.put("region", TEST_REGION);
+    attributes.put("serviceName", "spinnaker-demo");
+    attributes.put("serviceId", "srv-v001");
+    attributes.put(
+        "serviceArn",
+        "arn:aws:servicediscovery:region:aws_account_id:service/srv-utcrh6wavdkggqtk");
+
+    DefaultCacheResult testResult = buildCacheResult(attributes, testNamespace, clusterKey);
+    ecsCache.addCacheResult("TestAgent", Collections.singletonList(testNamespace), testResult);
+
+    // when
+    Response response = get(url).then().contentType(ContentType.JSON).extract().response();
+    log.info("SERVICE REGISTRY RESPONSE: " + response.asString());
```

Comment:
let's remove these log statements - they're useful for debug/testing but should be omitted from the final tests

Reply:
modification is done.